test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <TrimmerRootAssembly Include="OpenTelemetry.Instrumentation.Runtime" />
     <TrimmerRootAssembly Include="OpenTelemetry.Instrumentation.EventCounters" />
+    <TrimmerRootAssembly Include="OpenTelemetry.Instrumentation.StackExchangeRedis" />
```

Comment:
Could you also update the [ci.yml](https://github.com/CodeBlanch/opentelemetry-dotnet-contrib/blob/main/.github/workflows/ci.yml#L271-L279) file to add a contains check for the key `redis` as well?

Similar to this change https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1406/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR289

Reply:
Thanks @utpilla! I just opened #1422 to try and smooth this out a bit.